Tests/SwiftParserTest/translated/RecoveryTests.swift

```diff
@@ -543,22 +602,20 @@ final class RecoveryTests: XCTestCase {
       """,
       diagnostics: [
         DiagnosticSpec(message: "expected Sequence expression for for-each loop")
-      ]
-    )
-  }
-
-  func testRecovery43() {
-    assertParse(
-      """
-      // https://github.com/apple/swift/issues/48502
-      struct User { let name: String? }
-      """
+      ],
+      fixedSource: """
+        for i in {
```

Comment:
This should be inserting a `<#expression#>` placeholder after `in`.

Reply:
https://github.com/apple/swift-syntax/issues/1602